### PR TITLE
Feature/tanks graph

### PR DESCRIPTION
**Pull request by Andileh**
Creating a Tank Level graph

### CODE REVIEW

File: waterlog/src/Components/SensorsTable.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import {  fetchSensors } from '../actions/SensorsActions';
+import { formatDate, getStatusIcon } from './../utils'
+import Loader from './Loader';
+
+class SensorTableComponent extends Component {	
+	componentDidMount() {
```
Comment (FredJEhlers) [inline waterlog/src/Components/SensorsTable.js:9]: Your indentation is all over the place please be consistent. 
Here you used 2 `waterlog/src/Components/DailyTankGraph.js`
and in this file you used clearly 4 tabs

Comment (FredJEhlers) [reply waterlog/src/Components/SensorsTable.js:9]: Please look at all the files you committed

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

File: waterlog/src/Components/SensorsTable.js (hunk 1 above)
Comment (Andileh) [reply waterlog/src/Components/SensorsTable.js:9]: I've changed all the files  to be 2 tabs

File: waterlog/src/Components/TankComponent.js [hunk 2]
```diff
@@ -7,41 +7,41 @@ import Loader from './Loader';
...
+        </p>
 				</div>
 			);
 		} else if (level.percentageLevel >= 41 && level.percentageLevel <= 79) {
 			return (
 				<div>
-					<img src={images.tanks_orange_new} className="image" alt="tank" />
-					<p id="percentage">{level.percentageLevel}%</p>
+					<img src={images.tank_orange} className="image" alt="tank" />
+					<p id="percentage">{level.percentageLevel}%</p>'
```
Comment (FredJEhlers) [inline waterlog/src/Components/TankComponent.js]: remove `'` end of line

File: waterlog/src/Components/TankGraph.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ import { Line, defaults } from 'react-chartjs-2';
 const TankGraph = (props) => {
   let labelX = props.props.dailytankgraph.dataPoints.map(a => (new Date(a.x).getHours() + ":00"));
   let dataY = props.props.dailytankgraph.dataPoints.map(a => Math.round(a.y));
-  var start = new Date(Date.now());
+  /*var start = new Date(Date.now());
```
Comment (FredJEhlers) [inline waterlog/src/Components/TankGraph.js]: GC

File: waterlog/src/Components/TankGraph.js [hunk 4]
```diff
@@ -14,12 +14,12 @@ const TankGraph = (props) => {
...
     forecast.push(1*i + 80);
-  }
+  }*/
   
   let data = {
-    labels: labelX,
+    labels: ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12'],
     datasets: [
-      {
+      /*{
```
Comment (FredJEhlers) [inline waterlog/src/Components/TankGraph.js]: GC

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

File: waterlog/src/Components/TankGraph.js [hunk 5]
```diff
@@ -14,12 +14,12 @@ const TankGraph = (props) => {
   let forecast = [];
   for(let i = startTime; i <= endTime; i = i + 3600){
     forecast.push(1*i + 80);
-  }*/
+  }
   
   let data = {
-    labels: ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12'],
+    labels: ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12','13', '14', '15', '16', '17', '18', '19', '20', '21', '22', '23', '24'],
```
Comment (FredJEhlers) [inline waterlog/src/Components/TankGraph.js]: What is this for?

File: waterlog/src/Stylesheets/_wastage.scss [hunk 6]
```diff
@@ -7,7 +7,19 @@
 	color: #4F5B62;
 	size: 14pt;
 }
+.wastageTank{
 
+	background-color:#4F5B62;
+  margin: 20px;
```
Comment (FredJEhlers) [inline waterlog/src/Stylesheets/_wastage.scss]: Inconsistent indentations and add line between declarations

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

File: waterlog/src/Components/TankGraph.js (hunk 5 above)
Comment (Andileh) [reply waterlog/src/Components/TankGraph.js]: it was for putting labels in the graph (x axis )   which must end up to 24 not 12

File: waterlog/src/Components/TankComponent.js (hunk 2 above)
Comment (Andileh) [reply waterlog/src/Components/TankComponent.js]: done

File: waterlog/src/Components/TankGraph.js (hunk 3 above)
Comment (Andileh) [reply waterlog/src/Components/TankGraph.js]: done

File: waterlog/src/Components/TankGraph.js (hunk 4 above)
Comment (Andileh) [reply waterlog/src/Components/TankGraph.js]: done

File: waterlog/src/Stylesheets/_wastage.scss (hunk 6 above)
Comment (Andileh) [reply waterlog/src/Stylesheets/_wastage.scss]: Done

**Review by FredJEhlers — APPROVED** [review]

File: waterlog/src/Components/TankGraph.js (hunk 5 above)
Comment (Andileh) [reply waterlog/src/Components/TankGraph.js]: I have remove  it was for testing purposes now its get data in the database

**Pull request merged**